dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/xml/internal/DOMBasedLibertyBuildPluginConfigurationBuilder.java

```diff
@@ -73,6 +104,39 @@ public void addToModel(Element elem) {
         }
     }
 
+    private void handleApplications(Element elem) {
+        // TODO For multiple Applications support, simply call checkChildElements(elem);
+        //       <applications>
+        //         <application>
+        NodeList children = elem.getChildNodes();
+        int numOfChildren = children.getLength();
+        for (int i = 0; i < numOfChildren; i++) {
+            Node child = children.item(i);
+            if (child != null && child.getNodeType() == Node.ELEMENT_NODE && child.getNodeName().equals(Constants.APPLICATION_ELEMENT)) {
+                checkChildElements((Element) child);
```

Comment:
Why is this different than handleServers?

Reply:
Should be the same, since they both have similar hierarchies.  Originally added the handleServer method to try out adding multi-server support.